sys/mips/mips/elf_machdep.c

```diff
@@ -70,7 +70,7 @@ struct sysentvec elf64c_freebsd_sysvec = {
 	.sv_sigcode	= cheri_sigcode,
 	.sv_szsigcode	= &szcheri_sigcode,
 #endif
-	.sv_name	= "CheriABI ELF64",
+	.sv_name	= "FreeBSD ELF64C",
```

Comment:
I'd like to string cheriabi to appear somewhere...

Reply:
So I'm not sure how that will be relevant when cheriabi is the native ABI.  It won't be a special ABI, just the native ABI at that point.  (We should probably start making uname return "mips64c" if it isn't already for the native ABI as well at some point.)  We could maybe call it "FreeBSD ELF64CHERI" but that seems a bit verbose given we are using just 'c' for MACHINE_ARCH.

Reply:
It's probably fine if it's just a comment in the places that implement it (somewhere in this file and imgact_elf64c.c).  We'll eventually want to write something down that's comperable of the SysV i386 ABI.

Reply:
Ok, I've added some comments here and in imgact_elf64c.c, though the latter isn't used (and no one probably looks at it in practice).